php_phongo.c

```diff
@@ -3195,6 +3199,17 @@ PHP_MINFO_FUNCTION(mongodb)
 	php_info_print_table_row(2, "libmongoc compression", "disabled");
 #endif
 
+#ifdef MONGOC_ENABLE_CLIENT_SIDE_ENCRYPTION
+#ifdef HAVE_SYSTEM_LIBMONGOCRYPT
+	php_info_print_table_row(2, "libmongocrypt headers version", MONGOCRYPT_VERSION);
+	php_info_print_table_row(2, "libmongocrypt library version", mongocrypt_version());
+#else
+	php_info_print_table_row(2, "libmongocrypt bundled version", MONGOCRYPT_VERSION);
+#endif
```

Comment:
Feel free to add libmongocrypt-specific crypto info here once you bump to 1.0.1 for MONGOCRYPT-226.

Reply:
Restored previous crypto output 👍